src/centralserver/admin-ui/src/views/Members/MemberList.vue

```diff
@@ -30,15 +30,15 @@
         <div class="xrd-view-title">{{ $t('members.header') }}</div>
         <xrd-search v-model="search" />
       </div>
-      <xrd-button data-test="add-member-button" @click="() => {}">
+      <xrd-button data-test="add-member-button">
         <xrd-icon-base class="xrd-large-button-icon"
           ><xrd-icon-add
         /></xrd-icon-base>
 
         {{ $t('members.addMember') }}</xrd-button
       >
     </div>
-
+    numero: {{ getCount }}
```

Comment:
Would this be a leftover from some intermediate draftings?  To be removed or localized...

Reply:
Yes, this was never meant to be pushed in git.